src/dart_diagnostic_provider.ts

```diff
@@ -17,7 +17,7 @@ export class DartDiagnosticProvider {
 
 	private handleErrors(notification: as.AnalysisErrorsNotification) {
 		let errors = notification.errors;
-		if (!getConfig('showTodos'))
+		if (!getConfig<boolean>('showTodos'))
```

Comment:
I'd also been putting config names into constants; though currently they're littered around the place and probably should go together somewhere; I'll probably pull them all together soon into a strongly-typed class.